packages/ui/src/views/repo/pull-request/details/components/conversation/sections/pull-request-changes-section.tsx

```diff
@@ -9,10 +11,65 @@ import {
 } from '@/views'
 import { cn } from '@utils/cn'
 import { getInitials } from '@utils/stringUtils'
+import { PanelAccordionShowButton } from '@views/repo/pull-request/details/components/conversation/sections/panel-accordion-show-button'
 import { isEmpty } from 'lodash-es'
 
 import { LineDescription, LineTitle } from './pull-request-line-title'
 
+const getStatusIcon = (status: string) => {
+  switch (status) {
+    case 'pending':
+      return <Icon name="pending-clock" className="text-icons-alert" />
+    case 'warning':
+      return <Icon name="triangle-warning" className="text-tertiary-background" />
+    case 'error':
+      return <Icon name="triangle-warning" className="text-destructive" />
+    default:
+      return <Icon name="success" className="text-foreground-success" />
+  }
+}
+
+interface HeaderItemProps {
+  header: string
+}
+
+const HeaderItem: FC<HeaderItemProps> = ({ header }: HeaderItemProps) => {
```

Comment:
`FC` gives you the type already
```suggestion
const HeaderItem: FC<HeaderItemProps> = ({ header }) => {
```